Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -904,7 +903,8 @@ protected async Task RegisterResources()
             ChapterManager = new ChapterManager(LibraryManager, LoggerFactory, ServerConfigurationManager, ItemRepository);
             RegisterSingleInstance(ChapterManager);
 
-            RegisterMediaEncoder(assemblyInfo);
+            MediaEncoder = new MediaBrowser.MediaEncoding.Encoder.MediaEncoder(LoggerFactory, JsonSerializer, StartupOptions.FFmpegPath, StartupOptions.FFprobePath, ServerConfigurationManager, FileSystemManager, () => SubtitleEncoder, () => MediaSourceManager, ProcessFactory, 5000);
```

Comment:
can you wrap long line?

Reply:
I can but followed the other long lines.  Will change.